docs/manual/developer_guide.adoc

```diff
@@ -227,6 +227,208 @@ The maintainer should check Bohdi packages for that update for positive/negative
+shared content, documentation, READMEs, Licenses, build files/configuration, etc.
+
+For example:
+
+[source,bash]
+----
+$ ls scap-security-guide/
+
+build
+BUILD.md
+chromium
+cmake
+CMakeLists.txt
+Contributors.md
+Contributors.xml
+debian8
+DISCLAIMER
+Dockerfile
+docs
+fedora
+firefox
+jboss_eap6
+jboss_fuse6
+jre
+LICENSE
+opensuse
+oval.config.in
+README.md
+rhel6
+rhel7
+rhevm3
+rhosp7
+shared
+sle11
+sle12
+tests
+ubuntu14
+ubuntu16
+webmin
+wrlinux
+----
+
+===== Important Top Level Directory Descriptions
+
+|===
+|Directory |Description
+
+|```build```
+| Can be used to build the content using CMake.
+
+|```cmake```
+| Contains the CMake build configuration files.
+
+|```docs```
+| Contains the Markdown Manuels, MAN pages, etc.
+
+|```shared```
+| Contains content, tools and utilities, scripts, etc. that can be used for
+multiple products.
+
+|```tests```
+| Contains the test suite for content validation and testing.
+|===
+
+The remaining directories such as `fedora`, `rhel7`, etc. are product
+directories.
+
+===== Important Top Level File Descriptions
+
+|===
+|File |Description
+
+|```BUILD.md```
+| Contains the content build instructions
+
+
+|```CMakeLists.txt```
+| Top-level CMake build configuration file
+
+|```Contributors.md```
+| *DO NOT MANUALLY EDIT* script-generated file
+
+|```Contributors.xml```
+| *DO NOT MANUALLY EDIT* script-generated file
+
+|```DISCLAIMER```
+| Disclaimer for usage of content
+
+|```Dockerfile```
+| CentOS7 Docker build file
+
+|```LICENSE```
+| Content license
+
+|```oval.config.in```
+| _Deprecated in future releases:_
+Build configuration for mapping product
+version to correspond `multi_platform` product tags
+
+|```README.md```
+| Project README file
+
+|===
+
+==== Product Structure/Layout
+
+When creating a new product, use the guidelines below for the directory layout:
+
+* *Do not* use capital letters
+* If product versions are required, use major versions only. For example,
+`rhel7`, `ubuntu16`, etc.
+* If the content to be produced does not matter on versions, *do not* add version
+numbers. For example: `fedora`, `firefox`, etc.
+* In addition, use only a maxdepth of 3 directories.
+
+Following these guidelines help with the usability and browsability of
+using and navigating the content.
+
+For example:
+[source,bash]
+----
+$ tree -d rhel7
+
+rhel7
+├── checks
+│   └── oval
+├── cpe
+├── fixes
+│   ├── ansible
+│   └── bash
+├── kickstart
+├── overlays
+├── profiles
+├── templates
+│   ├── csv
+├── transforms
+└── utils
+
+13 directories
+----
+
+===== Product Level Directory Descriptions
+
+|===
+|Directory |Description
+
+|```checks```
+|```[red]#Required#``` Contains content such as OVAL to check whether or not a
+system is configured correctly to meet government or commercial compliance
+standards. Can contain the following directories: ```oval```
+
+|```cpe```
+|```[red]#Required#``` Contains the Common Platform Enumeration (CPE) product
```

Comment:
If I view it on github it isn't red, it's typewrited

Reply:
@jan-cerny I believe (checking the cheatsheet) that GitHub doesn't support color. The color is more so for asciidoc. I think how it shows up on GH is fine.